.github/workflows/changelog_generation.yml

```diff
@@ -0,0 +1,38 @@
+name: Compile changelogs
+
+on:
+  push:
+    branches:
+    - dev
+
+jobs:
+  Compile_Changelog:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Where if: github.repository == 'NebulaSS13/Nebula'

Reply:
checked test script, I think this not needed to work